upgrades/upgrade_testing_framework/clients/rest_client_default.py

```diff
@@ -0,0 +1,105 @@
+import json
+
+from upgrade_testing_framework.clients.rest_client_base import RESTClientBase
+import upgrade_testing_framework.clients.rest_ops as ops
+
+class RESTClientDefault(RESTClientBase):
```

Comment:
Does it make sense here to have a constructor here that accepts a host and port and then creates the base RESTPath and stores it, for these methods to simply append to with some functionality or is the goal here to not be stateful to any specific cluster

Reply:
Not sure it really matters much in the grand scheme of things, but I see your point about the client encapsulating a reference to the entity it is communicating with.  I'm fine w/ moving to that in the future if we want to.

Reply:
Not really, and not a huge preference from my side just a thought

Reply:
I've thought about this a bit more and think I came around to your point of view.  It's less arguments being passed around to have the client encapsulate the prefix and port from the RESTPath object, and we can make multiple instances of a RESTClient to talk to different servers.  Will keep it in mind for future revisions.